### DISCUSSION TITLE
Calculating wind stress

### DISCUSSION THREAD

**Discussion by sarriegi** [NOAA-EMC/WW3#980] [Q&A]
Hello, from what I know the wind stress is not available as output in WW3. What is the most correct way of calculating wind stress from on of the output variables (wind speed, or friction velocity maybe)? Is it correct to assume that the wind stress τ is: τ=ρu*u*
where ρ is the air density and u* is the friction velocity?

Angelos

@aliabdolali

**Comment by Biao-Zhao** [comment]
To my knowledge, the wind stress is   τ=ρu*u* or  τ=ρCdu10u10. hope this can help you
